packages/@orbit/data/src/query-term.ts

```diff
@@ -202,7 +202,7 @@ function filterParamToSpecifier(param: FilterQBParam): FilterSpecifier {
         records: (param as RelatedRecordsFilterQBParam).records
       } as RelatedRecordsFilterSpecifier;
     }
-  } else {
+  } else if ((param as any).attribute) {
```

Comment:
will it work if you do like in the rest of the method `param.hasOwnProperty('attributes')`?

Reply:
Yep. Updated. Thanks!